exercises/practice/yacht/.docs/instructions.md

```diff
@@ -0,0 +1,35 @@
+# Instructions
+
+The dice game [Yacht][yacht] is from the same family as Poker Dice, Generala and particularly Yahtzee, of which it is a precursor.
+In the game, five dice are rolled and the result can be entered in any of twelve categories.
+The score of a throw of the dice depends on category chosen.
+
+## Scores in Yacht
+
+| Category        | Score                  | Description                              | Example             |
+| --------------- | ---------------------- | ---------------------------------------- | ------------------- |
+| Ones            | 1 × number of ones     | Any combination                          | 1 1 1 4 5 scores 3  |
+| Twos            | 2 × number of twos     | Any combination                          | 2 2 3 4 5 scores 4  |
+| Threes          | 3 × number of threes   | Any combination                          | 3 3 3 3 3 scores 15 |
+| Fours           | 4 × number of fours    | Any combination                          | 1 2 3 3 5 scores 0  |
+| Fives           | 5 × number of fives    | Any combination                          | 5 1 5 2 5 scores 15 |
+| Sixes           | 6 × number of sixes    | Any combination                          | 2 3 4 5 6 scores 6  |
+| Full House      | Total of the dice      | Three of one number and two of another   | 3 3 3 5 5 scores 19 |
+| Four of a Kind  | Total of the four dice | At least four dice showing the same face | 4 4 4 4 6 scores 16 |
+| Little Straight | 30 points              | 1-2-3-4-5                                | 1 2 3 4 5 scores 30 |
+| Big Straight    | 30 points              | 2-3-4-5-6                                | 2 3 4 5 6 scores 30 |
```

Comment:
Not sure if little and big are commonly used or "small" and "large" is more commonly used.  (Personal experience playing for decades with a small sample of people, rather than a globally applied knowledge.)

Reply:
@kotp I'm also more familiar with the "small" and "large" terms from personal experience, but I was just going off what was defined in the canonical data file: https://github.com/exercism/problem-specifications/blob/main/exercises/yacht/canonical-data.json#L9

Reply:
That is fine.  Noticed it so brought it up here.  We can choose to change it for this track specifically or leave it as a potential update from Problem Specifications at some point in the future.

You are doing the work, so I leave it to you.

If you agree with the change, though, feel free to talk about it and maybe someone will tackle the change.